haiku-apps/beaccessible/beaccessible-1.0.0.recipe

```diff
@@ -0,0 +1,46 @@
+SUMMARY="An Access-like front-end to SQLite"
+DESCRIPTION="BeAccessible, an Access-like front-end to the SQLite database."
+HOMEPAGE="https://github.com/HaikuArchives/BeAccessible"
+COPYRIGHT="2004 randar"
+LICENSE="MIT"
+REVISION="1"
+COMMIT="3d9f8ff6b106375680422ab155bcb1b0815c9884"
+SOURCE_URI="https://github.com/HaikuArchives/BeAccessible/archive/$COMMIT.tar.gz"
+CHECKSUM_SHA256="097a5b5556353336bbb2772879c94d57e5f2e952cb31d1d5dcf62b358ee30701"
+SOURCE_DIR="BeAccessible-$COMMIT"
+
+ARCHITECTURES="x86_gcc2 !x86 !x86_64"
+
+PROVIDES="
+	beaccessible = $portVersion
+	app:BeAccessible = $portVersion
+	"
+REQUIRES="
+	haiku
+	lib:libexpat
+	lib:libsqlite3
+	"
+
+BUILD_REQUIRES="
+	haiku_devel
+	devel:libexpat
+	devel:libsqlite3
+	"
+BUILD_PREREQUIRES="
+	makefile_engine
+	cmd:g++
+	cmd:make
+	"
+
+BUILD()
+{
+	cd src
+	make $jobArgs OBJ_DIR=objects
+}
+
+INSTALL()
+{
+	mkdir $appsDir
```

Comment:
I think it is 
It's safer to use "mkdir -p".
Well, it is even better to use install:
```diff
-	mkdir $appsDir
+	install -d $appsDir
-	cp src/objects/BeAccessible $appsDir
+	install -t $appsDir src/objects/BeAccessible
```
I know the result is the same. But by using install we have more flexibility. Image we decide to do something (e.g. add a signature to each runtime we install) then we could modify `cmd:install` to do that, without having to change those recipes which already use install.

Reply:
How should an add-on binary like Archiver be installed? I suppose it would be different than a regular executable like BeAccessible.

Reply:
There is an $addOnsDir. Some examples of use (found right now):
* https://github.com/haikuports/haikuports/tree/master/haiku-apps/konfetti
* https://github.com/haikuports/haikuports/tree/master/app-emulation/virtualbox-guest-additions

Reply:
Thanks!